server/sheets.go

```diff
@@ -118,13 +118,28 @@ func prune(dirs []*Dir) []*Dir {
 	return dirs
 }
 
+func hasDeflate(acceptEncoding []string) bool {
+	for _, s := range acceptEncoding {
+		for _, encoding := range strings.Split(s, ", ") {
```

Comment:
Probably should not rely on there being a space after the comma and instead split just on the comma and then use strings.TrimSpace() to checking for deflate.